src/Rings/MPolyMap/MPolyRing.jl

```diff
@@ -69,13 +70,20 @@ julia> Qi, i = quadratic_field(-1)
 julia> S, (x, y) = polynomial_ring(Qi, ["x", "y"]);
 
 julia> G = hom(S, S, hom(Qi, Qi, -i), [x^2, y^2])
-Map with following data
-Domain:
-=======
-Multivariate polynomial ring in 2 variables over imaginary quadratic field defined by x^2 + 1
-Codomain:
-=========
-Multivariate polynomial ring in 2 variables over imaginary quadratic field defined by x^2 + 1
+Ring homomorphism
+  from multivariate polynomial ring in 2 variables over imaginary quadratic field defined by x^2 + 1
+  to multivariate polynomial ring in 2 variables over imaginary quadratic field defined by x^2 + 1
+defined by
+  x -> x^2
+  y -> y^2
+with map on coefficients
+  Map with following data
+    Domain:
+    =======
+    Qi
+  Codomain:
+    =========
+    Qi
```

Comment:
There is something weird going on with indentation here.

Reply:
It is exactly as intended. The problem is just that the map on coefficients does not follow the new printing guidelines.  There is a PR in AbstractAlgebra addressing this.

Reply:
I would have expected `Domain` and `Codomain` to be in the same level

Reply:
You are right, sorry.